### PR TITLE
Added support to split invitation emails #133

### PR DESCRIPTION
**Pull request by JoseAntonioTorre**
Support for adding emails as tags.
Now you can paste an email list of comma-separated, tab-separated, semicolon-separated
and the input will render each email as a tag.
Also, you can enter emails in the input and use any of those characters in order to create an email tag. Finally, if you enter an email and click enter it will create a tag.
I have added support to not allow duplicates.
image

### CODE REVIEW

File: ihatemoney/static/css/tagsinput.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+.tagsinput,.tagsinput *{box-sizing:border-box}
```
Comment (almet) [inline ihatemoney/static/css/tagsinput.css]: Can you put these instructions on multiple lines, rather than having each selector on a distinct line?

Comment (JoseAntonioTorre) [reply ihatemoney/static/css/tagsinput.css]: I'm used to be like you and my mind has changed (I don't know if in the correct direction). I think that the functionality that this library (plugin of jquery) implements it is to easy to develop and in fact, I have read all the code in order to make sure that I can implement the behaviour that you told me. The code is straightforward and I think that the dependency should not be a problem (the code can be maintained by ourself).

This afternoon I am going to correct the css and notify you when ready.

**Comment by JoseAntonioTorre** [comment]
@almet Ready!

**Comment by almet** [comment]
Thank you :-)

**Pull request merged**